index.html

```diff
@@ -6,17 +6,44 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   </head>
   <body>
-    <form>
-      <input type="text" id="slug" />
-      <input type="submit" value="Slug it" onclick="slugIt(); return false;"></input>
+    <form id="form" onSubmit="" style="max-width:60ch">
+			<fieldset>
+
+				<label>Input text</label>
+				<input type="text" id="input" value="An Apple computer" />
+			</fieldset>
+
+
+
+			<fieldset>
+				<label for="replacement">Replacement character</label>
+				<input type="text" id="replacement" name="replacement" value="-" />
+			</fieldset>
+
+
+			<fieldset>
+			<label>
+				<input type="checkbox" name="lowercase" checked /> Lowercase
```

Comment:
This doesn't work. Whether the checkbox is checked or unchecked, the result is lowercased. That's because the default value is `true`. So the JS code will need to explicitly set it to `false` if the box is unchecked.

Reply:
Oh, wait, the JS code looks fine. But for some reason, it's not working for me testing locally. I wonder why.

Reply:
Now it's working. I must have had something cached or something weird like that. Sorry about the noise.